iModelJsNodeAddon/api_package/ts/src/NativeLibrary.ts

```diff
@@ -521,8 +521,8 @@ export declare namespace IModelJsNative {
     public hasPendingTxns(): boolean;
     public hasUnsavedChanges(): boolean;
     public importFunctionalSchema(): DbResult;
-    public importSchemas(schemaFileNames: string[]): DbResult;
-    public importXmlSchemas(serializedXmlSchemas: string[]): DbResult;
+    public importSchemas(schemaFileNames: string[], allowDataTransformDuringSchemaUpdate: boolean): DbResult;

```

Comment:
if you're going to change the signature of these methods, why not a new "options" arg?

Reply:
Good point. I will instead use options.